packages/core/src/agent/EnvelopeService.ts

```diff
@@ -29,38 +30,44 @@ class EnvelopeService {
   }
 
   public async packMessage(payload: AgentMessage, keys: EnvelopeKeys): Promise<EncryptedMessage> {
-    const { routingKeys, senderKey } = keys
-    let recipientKeys = keys.recipientKeys
+    const { recipientKeys, routingKeys, senderKey } = keys
+    let recipientVerkeys = recipientKeys.map(didKeyToVerkey)
```

Comment:
I would rather not re-introduce the term verkey into more places of the codebase. It feels to me like something that is left-over from indy and hides what the value actually is: a base58 encoded public key (took me really long to understand a verkey is just that).

Maybe we can instead start introducing the encoding in the variable name?

So `senderPublicKeyBase58`, etc... What do you think?

Reply:
Agree, although `senderPublicKeyBase58` sems terrible but it's better naming. What about just `senderKeyBase58`? There are no private keys in other places. Just idea.

Reply:
Yeah sure `senderKeyBase58` sounds fine to me, it's explicit enough